models/orca_whirlpool/orca_whirlpool_trades.sql

```diff
@@ -0,0 +1,177 @@
+ {{
+  config(
+        tags = ['dunesql'],
+        schema = 'orca_whirlpool',
+        alias = alias('trades'),
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index', 'tx_index', 'project_program_id'],
```

Comment:
you're 100% certain these fields are what makes a row unique?

Reply:
one issue we've had in the past is `null` values as one of the join conditions, where the join doesn't return as expected (i.e. the join in the merge statement). try to find any nulls in those fields and do `coalesce(<field>, -1)` to ensure everything has a value across all unique keys. of course, match the datatype in the coalesce, i just picked `-1` randomly

Reply:
this was in fact it! so odd. okay so it's working fine for 7 days, how do we want to proceed with the merge?